.github/workflows/comprehensive-testing.yml

```
@@ -1,1121 +1,26 @@
-name: Comprehensive Testing Pipeline
-
-permissions:
-  contents: read
-  pull-requests: write  # Required to comment on PRs
-  issues: write         # Required to create issues for failures
-  checks: write         # Required to create check runs
-  actions: read         # Required to download artifacts
+# DEPRECATED: This workflow has been replaced by post-merge-validation.yml
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)